docs/user/advanced.rst

```diff
@@ -697,8 +697,9 @@ Encodings
 When you receive a response, Requests makes a guess at the encoding to
 use for decoding the response when you access the :attr:`Response.text
 <requests.Response.text>` attribute. Requests will first check for an
-encoding in the HTTP header, and if none is present, will use `chardet
-<https://pypi.org/project/chardet/>`_ to attempt to guess the encoding.
+encoding in the HTTP header, and if none is present, will use
+`charset_normalizer <https://pypi.org/project/charset_normalizer/>`_ to attempt
+to guess the encoding.
```

Comment:
I would add a bit of explanation there. 

- What happens if chardet is installed
- Keep both lib name as python 2 is keeping chardet by default

Reply:
Updated.